### ISSUE TITLE
Alexa Auth

### ISSUE THREAD

**Issue by MaxLucien** [bbindreiter/node-red-contrib-alexa-remote2-applestrudel#181]
I have node_red running in a docker environment.
the authentication cookie is simply not stored.
I created an empty file and tried several paths.
e.g.:
/data/alexa_auth_cookie.txt

/data/ is the data directory of Node_Red

I'm starting to get desperate, I'm asking for help

**Comment by tribbles3** [comment]
Do you get an error message in node-red when you do the browser authentication? As far as I can tell, the cookie file is hidden. So if you don't get an error message when doing the authentication, the file is getting created.

**Comment by MaxLucien** [comment]
Node_Red reports that there are problems accessing the file.

I created the file in the data volume of the Docker container.
Node_Red itself can also access data here.
I also gave the file read/write permissions for everyone (0777). Last but not least, I added the cookie information manually into the file, but nothing helped.

However, it now works.

I reinstalled the extension several times, but the error still remained.

In the end I did something absurd for me and set the path of the file to ./alexa_auth_cookie.txt with proxy authentication and it worked.

Unfortunately, the description of the extension regarding the file path is rather poor.

However, perhaps my experience will help other users.

**Comment by MaxLucien** [comment]
Node_Red reports that there are problems accessing the file.

I created the file in the data volume of the Docker container.
Node_Red itself can also access data here.
I also gave the file read/write permissions for everyone (0777). Last but not least, I added the cookie information manually into the file, but nothing helped.

However, it now works.

I reinstalled the extension several times, but the error still remained.

In the end I did something absurd for me and set the path of the file to ./alexa_auth_cookie.txt with proxy authentication and it worked.

Unfortunately, the description of the extension regarding the file path is rather poor.

However, perhaps my experience will help other users.
